controllers/voteAnswers.js

```diff
@@ -18,7 +18,7 @@ const voteAnswers = async (req, res) => {
             });
             return res.status(200).json(vote);
         } else {
-            return res.status(404).json({ messages: err.message });
+            return res.status(500).json({ messages: err.message });
```

Comment:
The comment was to format the json response but not necessarily changing the status. Initially you had this `return res.status(404).json('answer was not found');` So what I wanted you to do is to return a proper JSON object which must have properties and their corresponding values as against what you had initially which was just a string. This is what I meant in essence
```javascript
return res.status(404).json({message:'answer was not found'});
```
When you are dealing with APIs, the response must always be in JSON format except in exceptional cases where it is required to be in other format.

Please take note so you would apply it in other instances.

Also,  what you have there currently will make the application to crash if the user supply `answer_id` that does not exist because you are making reference to a variable that is not define - `err.message`; `err` is not defined on or before that line 24. You can try it on postman.